osu.Game/Screens/SelectV2/BeatmapCarouselFilterGrouping.cs

```diff
@@ -395,6 +395,16 @@ private GroupDefinition defineGroupByCollection(BeatmapInfo beatmap, IEnumerable
             return new GroupDefinition(1, "Not in collection");
         }
 
+        private GroupDefinition defineGroupByOwnMaps(BeatmapInfo beatmap, int? localUserId, string? localUserUsername)
+        {
+            var author = beatmap.BeatmapSet!.Metadata.Author;
+
+            if (author.OnlineID == localUserId || author.Username == localUserUsername)
+                return new GroupDefinition(0, "My maps");
+
+            return new GroupDefinition(1, "Not my maps");
```

Comment:
This kinda spoils the point of this group in stable, which is that it only shows your own beatmaps. I dunno.

Basically with this present, you first need to spent 5-10 seconds figuring how to get to the "my maps" group, or why it's not working like stable and still showing all maps.

Is there a reason we can't just remove it?

Reply:
I will have to alter the grouping code a bit to allow discarding beatmaps, so I thought it's fine to just show it in a separate group. I'll change it.

Reply:
Yeah I think it's best to match stable for usability reasons here, even though I can understand why you implemented the way you did.

Reply:
Applied in https://github.com/ppy/osu/pull/34539/commits/5c9f2a0402733385597812676effa48a3c78a84a